wapitiCore/main/wapiti.py

```diff
@@ -876,8 +885,15 @@ def wapiti_main():
 
     parser.add_argument(
         "--max-scan-time",
-        metavar="MINUTES",
-        help=_("Set how many minutes you want the scan to last (floats accepted)"),
+        metavar="SECONDS",
+        help=_("Set how many seconds you want the scan to last (floats accepted)"),
+        type=float, default=0
+    )
+
+    parser.add_argument(
+        "--max-attack-time",
+        metavar="SECONDS",
+        help=_("Set how many seconds you want the attack to last (floats accepted)"),
```

Comment:
You should mention that this is the time for each module and not for the whole attack process.

Reply:
Otherwise it seems good to me

Reply:
I changed it to "Set how many seconds you want each attack mdoule to last (floats accepted)"

Reply:
I also added a PR for the wapiti man https://github.com/wapiti-scanner/wapiti-scanner.github.io/pull/1